packages/panels/src/Pages/Auth/PasswordReset/RequestPasswordReset.php

```diff
@@ -80,21 +80,19 @@ function (CanResetPassword $user, string $token): void {
             },
         );
 
-        if ($status === Password::RESET_THROTTLED) {
+        if ($status === Password::RESET_LINK_SENT) {
+            Notification::make()
+                ->title(__($status))
+                ->success()
+                ->send();
+        } else {
             Notification::make()
                 ->title(__($status))
                 ->danger()
                 ->send();
-
-            return;
         }
 
         $this->form->fill();
```

Comment:
Hasn't the behavior now changed since we're calling `fill()` even when throttled, @danharrin?

Reply:
good catch